src/main/java/com/hms/flexyosisoftconnector/dataserver/OSIsoftServer.java

```diff
@@ -193,9 +195,29 @@ public static boolean RequestHTTPS(
     if (res == SCHttpUtility.HTTPX_CODE_NO_ERROR) {
       success = true;
     }
+
+    appendTimeStampToFile(FileName);
     return success;
   }
 
+  /**
+   * Appends the current time to the end of a file for logging purposes.
+   *
+   * @param fileName the file path to append a time stamp to.
+   */
+  private static void appendTimeStampToFile(String fileName) {
+    long localTimeMillis = LocalTimeOffsetCalculator.getLocalTimeOffsetMilliseconds();
+    Date currentDate = new Date(localTimeMillis);
+    String currentTime = "\n" + currentDate;
+
+    try {
+      FileAccessManager.appendStringToFile(fileName, currentTime);
+    } catch (IOException e) {
+      Logger.LOG_WARN("Unable to add timestamp to response file.");
+      e.printStackTrace();

```

Comment:
Improper logging of exception. Exceptions should be output via `Logger.LOG_EXCEPTION(e);` as opposed to `e.printStackTrace()` which bypasses our logging system.